types/argparse/argparse-tests.ts

```diff
@@ -152,11 +153,19 @@ console.log('-----------');
 args = nargsExample.parse_args('--bar b c f --foo a'.split(' '));
 console.dir(args);
 
+function positiveInt(s: string): number {
+    const i = parseInt(s, 10);
+    if (i <= 0) {
+        throw new ArgumentTypeError('must be positive');
+    }
+    return i;
+}
+
 const parent_parser = new ArgumentParser({ add_help: false });
 // note add_help:false to prevent duplication of the -h option
 parent_parser.add_argument(
     '--parent',
-    { type: 'int', help: 'parent' }
+    { type: positiveInt, help: 'parent' }
```

Comment:
Could you make the test additive instead? Right now it's not clear whether this change is breaking or not from the test alone.